network/hive.go

```diff
@@ -249,13 +252,40 @@ func (h *Hive) loadPeers() error {
 		}
 	}
 	log.Info(fmt.Sprintf("hive %08x: peers loaded", h.BaseAddr()[:4]))
+	errRegistering := h.Register(as...)
+	var conns []*BzzAddr
+	err = h.Store.Get("conns", &conns)
+	if err != nil {
+		if err == state.ErrNotFound {
+			log.Info(fmt.Sprintf("hive %08x: no persisted peerc onnections found", h.BaseAddr()[:4]))
+		} else {
+			log.Warn(fmt.Sprintf("hive %08x: error loading connections: %v", h.BaseAddr()[:4], err))
+		}
+
+	} else {
+		go h.suggestInitialPeers(conns)
+	}
+	return errRegistering
+}
 
-	return h.Register(as...)
+func (h *Hive) suggestInitialPeers(conns []*BzzAddr) {
```

Comment:
this is not just suggestion like from the point of view of kademlia.
pls call it `connectPeers(conns  ...*BzzAddr)` and the argument